ObieWalker/EventsManager
# (chore): refactor code

## What does this PR do?
This PR implements changes to my code that remove console log errors and improve functionality.
 
## Description of Task to be completed?
- remove console errors
- make login and register page responsive
- fix failing tests as a result of refactoring
 
## How should this be manually tested?
* clone the repo and change into the project directory
* run npm install to install required packages
* create a .env file and add all the required environment variables
* start up the server with the command npm run start:dev
* start up the client side with the command npm run build:dev
* visit http://localhost:3000 and to load application in your browser
* sign up with a user account
* on your dashboard, peruse through the application to test functionalities
* open the browser console to see an errorless view.

## Any background context you want to provide?
N/A
## What are the relevant pivotal tracker stories?(if applicable)
 #157062544
